scripts/utils.ts

```diff
@@ -292,7 +292,7 @@ export async function checkInstanceHealth(instanceId: string, region: string = "
       return {
```

Comment:
In general in this file, we want to add the prefix to the error messages that are variable and aren't hard coded strings